app/policies/application_policy.rb

```diff
@@ -10,4 +10,23 @@ def update?
   def destroy?
     update?
   end
+
+
+  private
+
+  def authorized?(scope)
+    if account_id_was.present? && account_id_was != account_id
+      token&.authorized?(account_id, scope) && token&.authorized?(account_id_was, scope)
+    else
+      token&.authorized?(account_id, scope)
+    end
+  end
+
+  def account_id
+    resource.account_id
+  end
+
+  def account_id_was
+    nil
```

Comment:
Why nil?  (vs `resource.account_id_was`)

Reply:
Could go either way, I guess I was imagining that we shouldn't do this check by default but :shrug: